__tests__/validation.test.js

```diff
@@ -0,0 +1,55 @@
+import { MESSAGES } from "../src/constants/messages";
+import { ERRORS } from "../src/constants/errors";
+import {
+  validateNameInput,
+  validateDescriptiontInput,
+  validateSelectInput,
+} from "../src/validation/validator";
+
+describe("모달 사용자 입력 테스트", () => {
+  describe("이름 입력 테스트", () => {
+    it("이름의 입력 값은 공백일 수 없다.", () => {
+      // given
+      const input = "";
+
+      // then
+      expect(() => validateNameInput(input)).toThrow(ERRORS.EMPTY_NAME);
```

Comment:
```suggestion
// given
const input = "";

// when
const validateAction = () => validateNameInput(input);

// then
expect(validateAction).toThrow(ERRORS.EMPTY_NAME);

```

너무 간단한 검증이고 반영될 필요는 없지만 공통적으로 given-when-then의 사고는 유지하는게 좀 더 좋을 것 같다는 개인적인 의견이에요